Makefile

```diff
@@ -25,14 +25,27 @@ GIT_COMMIT := $(shell git rev-parse HEAD 2> /dev/null || true)
 
 all: binary
 
-binary:
+binary: ${DEST}skopeo
+
+# Build a docker image (skopeobuild) that has everything we need to build.
+# Then do the build and the output (skopeo) shold appear in ${DEST} dir (or .).
+${DEST}skopeo: cmd/skopeo
+	docker build ${DOCKER_BUILD_ARGS} -f Dockerfile.build -t skopeobuildimage .
```

Comment:
Thanks, I forgot about the cache. Yes, the cache makes this cheap, and with the cache removing the container instead of permanently polluting the namespace is nice. Never mind.